### ISSUE TITLE
clicking on the applet makes grey rectangle appear

### ISSUE THREAD

**Issue by n1trux** [cedl38/xfce4-windowck-plugin#21]
I have the following problem: If I click on the applet, a grey rectangle appears on desktop.

![screenshot](https://cloud.githubusercontent.com/assets/7255542/8016134/da6403be-0bdf-11e5-9d0b-0df76b11ef26.jpg)

How can I debug this?

**Comment by cedl38** [comment]
Hello,

Thanks for reporting this issue. Actually, I've no idea what this append. Could you give me more detail about this issue? In which case the issue append, etc..  

Thanks,

**Comment by n1trux** [comment]
I found the solution, sort of. I had a second monitor (my TV) attached to HDMI, which looked like this in arandr: 
![arandr](https://cloud.githubusercontent.com/assets/7255542/8093900/05152efc-0fc4-11e5-8ebe-605cd70d6538.png)

This is exactly the area which is covered by the grey rectangle. If I disable the second screen or if I'm mirroring my 1440p screen to the 1080p screen (downscaling, effectively), the grey rectangle disappears.

Do you need more information? If so, which?

**Comment by cedl38** [comment]
Does this issue appear only on desktop ?

2015-06-10 22:59 GMT+02:00 Crack Duck <notifications@github.com>:

**Comment by n1trux** [comment]
Yes, only visible on "xfdesktop", didn't try it with other desktops. I can only click it away if I mark the "Trash Bin" icon and then click on the grey area or the windowck-plugin in the panel.

**Comment by cedl38** [comment]
From the latest commit, the applet is now completely hidden on the panel, so interaction with mouse is disabled in this case.
This change should workaround for this issue when plugin is hidden on desktop (default behavior).
Could you tries it for those cases:
- plugin hidden on desktop
- plugin displayed on desktop

Regards,

**Comment by n1trux** [comment]
sadly, this is not fixed just yet. The rectangle appears black now:

![screenshot-blackrectangle](https://cloud.githubusercontent.com/assets/7255542/8148091/2717a4c0-128b-11e5-8b0f-16b639a5b139.jpg)

But that only happens when I select "clone displays" in `xfce4-display-settings`. If I manually disable the screen in arandr and enable it again, it works – but then my second screen with lower resolution doesn't show the whole – downscaled – image, but only the part which is represented as the black rectangle on my screenshots.

**Comment by cedl38** [comment]
Hello,

Thanks for your report.
Sory I missed your email. I see it just now. I have curently no idea why this problem come from...
